### DISCUSSION TITLE
[Shop] user enterable "donation" amount to trade for tokens

### DISCUSSION THREAD

**Discussion by ScaredDonut** [gameserverapp/Platform#654] [Suggestions]
Hello!

It be good to have the ability to sell tokens for x amount of dollars. So kinda like the donation suggestion but in this case you trade X amount of $ for X amount of coins/tokens.

So for example on the admin panel we'd have a page where it asks:

"How many tokens should a user get for each $1.00 they donate"

So lets say I chose 1000.

On the client side of the website they'd have a donation widget/page where it asks them how much they want to donate/credits they want to buy and if they enter 2.75 for example they'd get 2,750 credits.

Thank you :)

**Comment by mvaessen** [comment]
closing because it's the same as #918
